user_guide_src/source/outgoing/view_cells.rst

```diff
@@ -80,7 +80,15 @@ Controlled Cells
 
 .. versionadded:: 4.3.0
 
-Controlled Cells have two primary goals: to make it as fast as possible to build the cell, and provide additional logic and flexibility to your views, if they need it. The class must extend ``CodeIgniter\View\Cells\Cell``. They should have a view file in the same folder. By convention the class name should be PascalCase and the view should be the snake_cased version of the class name. So, for example, if you have a ``MyCell`` class, the view file should be ``my_cell.php``.
+Controlled cells have two primary goals: to make it as fast as possible to build the cell, and provide additional logic and
+flexibility to your views, if they need it. The class must extend ``CodeIgniter\View\Cells\Cell``. They should have a view file
+in the same folder. By convention, the class name should be in PascalCase suffixed with ``Cell`` and the view should be
+the snake_cased version of the class name, without the suffix. For example, if you have a ``MyCell`` class, the view file
+should be ``my.php``.
+
+.. note:: Prior to v4.3.5, the generated view file ends with ``_cell.php``. This was deemed incorrect and corrected accordingly.
+    For backward compatibility, existing view files ending with ``_cell.php`` will still be located and loaded. However, it is
+    recommended to update the filenames to remove the trailing ``_cell``.
```

Comment:
Is it needed to recommend to remove `_cell` in view files?
Do you have a plan to remove the logic to search `_cell` views in the future?

I think the current convention (`MyCell` → `my_cell.php`) should be also supported as ever.

Reply:
Let's have @lonnieezell have a say on this.

Reply:
Remember, people can also specify the view file to use within the cell class itself, so I don't think it's that big of a concern. 

Unless we have a plan to drop that support in the future, I don't know that there's much need to recommend them renaming the files. The performance gain would be unnoticeable.

Reply:
I hope CI4 supports both conventions, and does not recommend renaming the view files.

Reply:
I removed the recommendation. Please check if wording is now okay.

Reply:
@paulbalandan I'm okay. Thank you.